src/lib/modules/charts/components/model-modal/model-modal.component.ts

```diff
@@ -43,12 +50,16 @@ export class ModelModalComponent implements OnInit {
         this.getClimateModels();
     }
 
+    ngOnChanges(changes: SimpleChanges) {
```

Comment:
Why add SimpleChanges?
My understanding is that you use SimpleChanges if you want access to it's properties, but it is not used except by implicit typing here.

Reply:
Whoops, leftover from previous testing that was reverted. I can remove since its not used.